.github/workflows/ci.yml

```
@@ -61,3 +61,38 @@ jobs:
     - name: Documentation
       run: |
         cabal haddock
+
+  stack:
+      name: stack / ghc ${{ matrix.ghc }}
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          stack: ["2.7.5"]
+          ghc: ["9.2.3"]
+
+      steps:
+      - uses: actions/checkout@v2
+
+      - uses: haskell/actions/setup@v1.2
+        name: Setup Haskell Stack
+        with:
+          ghc-version: ${{ matrix.ghc }}
+          stack-version: ${{ matrix.stack }}
+
+      - uses: actions/cache@v2
```

The same as checkout, we can update it. Dependabot can update later but let's update straight away because we know newer versions already 🙂 
```suggestion
      - uses: actions/cache@v3
```